jsk_2016_01_baxter_apc/scripts/create_udev_rules

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Is this really needed?
Same as rosserial rules?
https://github.com/start-jsk/jsk_apc#optional